src/engine/controls/cuecontrol.cpp

```diff
@@ -135,6 +135,10 @@ CueControl::CueControl(const QString& group,
     m_pCueIndicator = new ControlIndicator(ConfigKey(group, "cue_indicator"));
     m_pPlayIndicator = new ControlIndicator(ConfigKey(group, "play_indicator"));
 
+    m_pIsPlaying = new ControlPushButton(ConfigKey(group, "is_playing"));
```

Comment:
I was also considering this, because it can also be used a display connection when blinking is not desired. 
However I am unsure if the name is unambiguous enough. Alternative:
"play_latched"  or "play_latched_indicator" depending if it is allowed to receive updates or not. 
What do you think?

Reply:
`play_latched` works for me.
I don't think this CO should blink. We should keep blinking to a minimum, also it would (visually) interfere with blinking Cue modes.